src/FSharpLint.Core/Rules/Conventions/VariableLengthGreaterThanOne.fs

```diff
@@ -0,0 +1,73 @@
+module FSharpLint.Rules.VariableLengthGreaterThanOne
+
+open System
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+
+let private checkIdentifierPart (identifier:Ident) (idText:string) =
+    let formatError errorName =
+        String.Format(Resources.GetString errorName, idText)
+
+    let addVariableOfLenghtOne (ident:Ident) = lazy(
```

Comment:
that's weird style, please place the lazy keyword in the next line

Reply:
Waa Woo. Done!